tests/www_rbac/api/experimental/test_endpoints.py

```diff
@@ -259,10 +269,16 @@ def setUpClass(cls):
 
     def setUp(self):
         super(TestPoolApiExperimental, self).setUp()
-        configuration.load_test_config()
-        app, _ = application.create_app(testing=True)
-        self.app = app.test_client()
-        self.session = Session()
+        conf.load_test_config()
+        self.app, self.appbuilder = application.create_app(session=Session, testing=True)
+        self.app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///'
```

Comment:
Repeated code from `test_endpoints.py` , maybe refactor as `get_rbac_flask_app()` in the base test class?

Reply:
Re-writing or copying chunks of code always calls for refactoring. DRY.

Reply:
thanks. Will update.